server/src/main/java/com/github/animelist/animelist/model/profilepage/StatisticsBlockAdditionalData.java

```diff
@@ -0,0 +1,52 @@
+package com.github.animelist.animelist.model.profilepage;
+
+public class StatisticsBlockAdditionalData {
+    private int entries;
```

Comment:
I think it may be a good idea to use the boxed implementation instead of primitives to represent null values correctly. It doesn't matter anyways if these are suppose to be non null anyways.